### PR TITLE
Fix mistake in proxy config

### PR DESCRIPTION
**Pull request by esalitrynskiy**
I can't handle how to use proxies with your config, it didn't work at all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -431,9 +431,15 @@ while not pointer is None:
 ```python3
 from instagram import WebAgent
 
-settings = {"proxies": {"any_ip": any_port}}
+settings = {
+    "proxies": {
+        'http': 'http://example.net:8888',
+        'https': 'https://example.net:8888'
+    }
+}
```
Comment (OlegYurchik) [inline README.md]: Is this description fit settings for "requests" and for "aiohttp"?

Comment (esalitrynskiy) [reply README.md]: No, sorry, but I've tested it only with "requests". Maybe your config works with "aiohttp".

Comment (esalitrynskiy) [reply README.md]:
```suggestion
agent = WebAgent(settings=settings)
```

But this string cause error anyway.

Comment (OlegYurchik) [reply README.md]: Thank you! If this work for requests - it is corect

**Pull request merged**